app/src/main/res/values-lt/strings.xml

```diff
@@ -0,0 +1,328 @@
+    <string name="profile_button_delete_success">Jūsų paskyra buvo sėkmingai ištrinta</string>
+    <string name="route_calendar">Kalendorius</string>
+    <string name="route_decorations">Dekoracijos</string>
+    <string name="route_friend">%1$s profilis</string>
+    <string name="route_friends">Draugai</string>
+    <string name="route_goals">Siekiai</string>
+    <string name="route_profile">Profilis</string>
+    <string name="route_settings">Nustatymai</string>
+    <string name="route_statistics">Statistikos</string>
+    <string name="route_survey">Apklausa</string>
+    <string name="screen_authentication_button_begin">Pradėkime!</string>
+    <string name="screen_authentication_toast_error">Nepavyko prisijungti. Bandykite dar kartą</string>
+    <string name="settings_button_reset">Atstatyti nustatymus</string>
+    <string name="settings_button_reset_description">Ar tikrai norite atstatyti visus savo nustatymus?</string>
+    <string name="settings_category_3rd_party_services_title">3-ųjų Šalių Paslaugos</string>
+    <string name="settings_category_appearance_title">Išvaizda</string>
+    <string name="settings_category_notifications_title">Pranešimai</string>
+    <string name="settings_locale_en">Anglų</string>
+    <string name="settings_locale_lt">Lietuvių</string>
+    <string name="settings_setting_cup_volume_description">Pasirinkite savo puodelio dydį tikslesniems vandens išgėrimo priminimams gauti. Įprastas nustatymas lygus %1$s ml</string>
+    <string name="settings_setting_cup_volume_title">Puodelio Dydis</string>
+    <string name="settings_setting_google_calendar_button_synchronize">Sinchronizuoti</string>
+    <string name="settings_setting_google_calendar_description">Susinchronizuokite savo \"Google\" kalendoriaus užduotis ir renginius su mūsų programėle, kad rastumėte visas savo veiklas vienoje vietoje. Praėjusių dienų \"Google\" užduotys nebus sinchronizuotos. Tik šiandienos ir ateities užduotys bus sinchronizuotos.</string>
+    <string name="settings_setting_google_calendar_synchronize_success">\"Google\" Kalendorius sėkmingai sinchronizuotas</string>
+    <string name="settings_setting_google_calendar_title">\"Google\" Kalendorius</string>
+    <string name="settings_setting_health_connect_button_configure">Konfigūruoti</string>
+    <string name="settings_setting_health_connect_description">Skirdami tam tikrus \"Health Connect\" leidimus padedate mums pateikti daugiau puikių funkcionalumų jums. Skirtus leidimus bet kada galite išjungti ateidami čia. \"Health Connect\" mums leidžia pasinaudoti kitų programėlių surinktais duomenimis apie jūsų aktyvumą.</string>
+    <string name="settings_setting_health_connect_title">\"Health Connect\" įrankis</string>
+    <string name="settings_setting_hydration_reminder_description">Pasirinkite ar norite įjungti vandens išgėrimo priminimus</string>
+    <string name="settings_setting_hydration_reminder_title">Vandens Išgėrimo Priminimas</string>
```

Comment:
Sounds a bit odd. Maybe "Vandens suvartojimo" ar "Vandens vartojimo" would be better? Unsure though.

Reply:
Decided to stay for `Gėrimo`